cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -197,6 +197,11 @@ func (s *initramfsMountsSuite) SetUpTest(c *C) {
 	restore = func() { dirs.SetRootDir("") }
 	s.AddCleanup(restore)
 
+	restore = main.MockWaitFile(func(string, time.Duration, int) error {
```

Comment:
should this function really be always mocked? in most cases we create/mock all the files before the initramfs-mounts routine that checks for the file is actually run, so I'm not sure if we need to have this mocked all the time, just mocked in the tests that manually verify this behavior

Reply:
My initial approach was just to create the file but that makes the PR quite a bit bigger, there are a lot of `/dev/disk/by-label/ubuntu-seed` paths not mocked right now because we did not look at the disk before in these tests. happy to explore this further, but definitely a followup :)

Reply:
Ah okay interesting I didn't realize that thanks for explaining